package/MDAnalysis/analysis/lineardensity.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+#     pylint: disable=E1101
+from __future__ import division
+
+import os.path as path
+import numpy as np
+
+from MDAnalysis.analysis.base import AnalysisBase
+
+class LinearDensity(AnalysisBase):
+    def __init__(self, universe, selection, grouping='atoms', binsize=0.25,
+                 start=None, stop=None, step=None):
+        self._ags = [selection] # allows use of run(parallel=True)
+        self._setup_frames(universe, start, stop, step)
+
+        self.binsize = binsize
+
+        # group of atoms on which to compute the COM (same as used in
+        # AtomGroup.wrap())
+        self.grouping = grouping
+
+        # Dictionary containing results
+        self.results = {'x': {'dim': 0}, 'y': {'dim': 1}, 'z': {'dim': 2}}
+        # Box sides
+        self.dimensions = [side for side in self._universe.dimensions[0:3]]
```

Comment:
This is just converting a np array into a list, just do dimensions[:3] to get the 3 lengths